### ISSUE TITLE
Add file chooser in the main method where the user can open a .txt file to generate random text.

### ISSUE THREAD

**Issue by JingMa87** [JingMa87/Random-Text-Generator#2]
I want the user to be able to open a file chooser, so they can load in a big text.

**Comment by MilindaLaknath** [comment]
Hi @JingMa87 

I can do this. I will update code.

Thank You.

**Comment by MilindaLaknath** [comment]
Hi @JingMa87 

@marcobarbosa16 already created GUI for the app. You should merge that pull request before I update the code. please let me know after you merge that.

Thank You.

**Comment by MilindaLaknath** [comment]
Hi @JingMa87 

Can I add action to "Generate" Button to generate random text using opened text file.

**Comment by JingMa87** [comment]
@MilindaLaknath Yes you can, but you should ask me that in the right Issue so we can keep track of who's doing what. 😅

**Comment by MilindaLaknath** [comment]
Thank you @JingMa87 
I added jFileChoser to GUI.java file. Isn't that you want.
